Database/Corrections/sodQuestFixes.lua

```diff
@@ -666,6 +666,16 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410096,
             [questKeys.zoneOrSort] = sortKeys.Shaman,
         },
+        [90032] = {
+            [questKeys.name] = "Lake Of Fire",
+            [questKeys.startedBy] = {{3537}},
+            [questKeys.finishedBy] = {{414658}},
```

Comment:
A note on this: Since it is a "fake quest" (a quest that never is in the quest log of a player), Questie won't show the finisher (since the quest will never be "accepted" and also never will be "ready for turn in").

So basically this field is irrelevant.

BUT it is good you have the ID of what is also related to this rune.

Therefore I suggest to add another fake quest and use this ID as starter. Give the quest the same name, but maybe change the objective text to make it more clear what to do at the different points.

I think I did the same for the "Enlightment" rune for Mages, which you get in Duskwood, but also need to visit two places without any quest.

Besides that: Your PR looks great, thank you already!

Reply:
Ill look into it tomorrow 😊

Reply:
Oh and a couple of follow-up questions:
1. For Enlightenment you put in [questKeys.requiredSpell] = -415942. Is this a way to make sure the fake quest "disappears" from the map/minimap once it is acquired? If so I'll do the same.
2. Do we have a good approach for selecting requiredLevel and questLevel? Many runes are not related to quests so they can be potentially be obtained at any level (even though it may involve navigating in higher level zones). IMO it makes sense to put requiredLevel to 1 and set an approximate questLevel depending on the required activity (e.g. navigating in an area with lvl 22 mobs in the case of Lake of Fire)

Reply:
Also maybe we should consider showing these fake rune quests regardless of player level. But maybe you guys already implemented it this way or for some reason decided not to?